covalent/executor/executor_plugins/remote_executor.py

```diff
@@ -40,6 +40,7 @@
     "poll_freq": 15,
     "remote_cache": ".cache/covalent",
     "credentials_file": "",
+    "covalent_version": "0.228.0-rc.0",
```

Comment:
Let's have the default as an empty string for now, we can later add a mechanism to determine the version automatically.